extensions/wikia/UserProfilePageV3/models/FavoriteWikisModel.class.php

```diff
@@ -46,62 +46,77 @@ public function __construct( User $user ) {
 	 * @return array
 	 */
 	private function getTopWikisFromDb( $limit = self::MAX_FAV_WIKIS ) {
-		global $wgCityId, $wgSpecialsDB;
+		global $wgDWStatsDB;
 
-		if ( empty( $wgCityId ) ) {
-			// staff/internal does not have stats db
-			$wikis = [];
-		} else {
-			$query = $this->getTopWikisQuery( $limit );
+		$dbr = wfGetDB(DB_SLAVE, [], $wgDWStatsDB );
+		$where = [
+			'period_id' => DataMartService::PERIOD_ID_WEEKLY,
+			'time_id >= NOW() - INTERVAL 90 DAY', // process only recent 90 days
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app